cmd/flightctl/main.go

```diff
@@ -31,20 +31,24 @@ const (
 )
 
 var (
-	resourceKinds = map[string]string{
+	clientConfigFile string
+	resourceKinds    = map[string]string{
 		"device":            "",
 		"enrollmentrequest": "",
 		"fleet":             "",
 		"repository":        "",
 		"resourcesync":      "",
 	}
 	resourceNameRegex = regexp.MustCompile(`^[a-zA-Z0-9\-]+$`)
-	serverUrl         = "https://localhost:3333"
 	fileExtensions    = []string{".json", ".yaml", ".yml"}
 	inputExtensions   = append(fileExtensions, "stdin")
 	legalOutputTypes  = []string{yamlFormat}
 )
 
+func init() {
+	clientConfigFile = filepath.Join(homedir.HomeDir(), ".flightctl", "client.yaml")
```

Comment:
question: would we want to allow the config path to also be defined by the user?

Reply:
We definitely should. Will add this in a follow-up PR.